components/bl-autocomplete-component/src/helpers/use-filtered-options.js

```diff
@@ -1,6 +1,6 @@
 import { useMemo } from 'react';
 
-const compareCaseInsensitive = (str1, str2) => str1.toLowerCase().includes(str2.toLowerCase());
+const compareCaseInsensitive = (str1 = '', str2 = '') => str1.toLowerCase().includes(str2.toLowerCase());
```

Comment:
```suggestion
const compareCaseInsensitive = (str1, str2) => str1?.toLowerCase().includes(str2.toLowerCase());
```

How about that? Check it please.